web/app/models/project_summary.ts

```diff
@@ -22,6 +22,6 @@ export class ProjectSummary {
     this.lane = projectSummary.lane;
     this.latestStatus = fastlaneStatusToEnum(projectSummary.latest_status);
```

Comment:
I think this should be undefined in the model instead of having the function return undefined. This way we can leverage tight type checks to ensure `fastlaneStatusToEnum` returns a build status and is given a fastlane build status.

This is similar to what you have for `latestDate`

Reply:
I misunderstood your previous comment my bad 😓